issuesync.rb

```diff
@@ -226,6 +229,14 @@ def updated_at() Time.parse(data['updated_at']) end
     def has_comments?() !data['comments'].zero? end
     def open?() data['state'] == 'open' end
     def closed?() !open? end
+
+    def labels()
+      s = ""
+      for label in data['labels']
+        s << "  _" + label['name'] + "_"
```

Comment:
Seems entirely reasonable to return the labels as an array, and certainly more consistent with the rest of the code. For formatting, I don't mind at all - I was trying to disambiguate from `[CLOSED]` - but that may be entirely unnecessary and having them the same might indeed be better.